custom_components/tahoma/select.py

```diff
@@ -21,7 +21,6 @@
 class OverkizSelectDescriptionMixin:
     """Define an entity description mixin for select entities."""
 
-    options: list[str | OverkizCommandParam]
```

Comment:
How does it work now?

Reply:
They added options to the native core description.